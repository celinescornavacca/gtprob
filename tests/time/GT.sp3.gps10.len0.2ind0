(((((B5,B6),B4),((B0,(B1,((B8,B9),B7))),(B2,B3))),(((C3,C4),C2),((C1,(C5,C6)),(C0,((C8,C9),C7))))),(((A1,(A2,(A7,(A8,A9)))),A0),(A3,((A5,A6),A4))));
