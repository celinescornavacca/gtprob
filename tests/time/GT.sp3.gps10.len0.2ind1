(((((B7,(B8,B9)),B0),(((B5,B6),B2),(B1,(B3,B4)))),((C0,(C4,C5)),((C1,(C6,C7)),((C3,(C8,C9)),C2)))),((A0,(((A8,A9),A7),(A3,A4))),((A2,(A5,A6)),A1)));
