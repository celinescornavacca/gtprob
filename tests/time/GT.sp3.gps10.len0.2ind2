((((B2,B3),((B0,((((B8,B9),B7),B4),B1)),(B5,B6))),(((C2,(C5,(C8,C9))),C0),(C1,((C3,C4),(C6,C7))))),((A0,(A8,A9)),(((A3,A4),(A5,(A6,A7))),(A1,A2))));
