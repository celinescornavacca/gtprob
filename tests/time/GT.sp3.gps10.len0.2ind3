((((B8,B9),((((B3,B4),B1),(((B6,B7),B5),B2)),B0)),(((((C4,(C7,(C8,C9))),C3),(C1,C2)),C0),(C5,C6))),(((((A4,A5),(A6,A7)),(A8,A9)),A0),((A2,A3),A1)));
