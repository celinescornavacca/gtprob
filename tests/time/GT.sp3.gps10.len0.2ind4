(((((A3,(A8,A9)),A2),A0),(A1,(A4,((A6,A7),A5)))),((((B6,B7),B0),(((B4,B5),B2),((B3,(B8,B9)),B1))),(((C1,(C4,C5)),C0),((C2,(C6,C7)),((C8,C9),C3)))));
