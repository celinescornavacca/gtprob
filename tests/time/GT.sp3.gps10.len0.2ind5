((((B0,B1),(((B4,B5),(B3,(B8,B9))),((B6,B7),B2))),(((C4,C5),C0),(((C6,C7),(C2,C3)),(C1,(C8,C9))))),(((A3,A4),(A1,A2)),(A0,(A5,((A8,A9),(A6,A7))))));
