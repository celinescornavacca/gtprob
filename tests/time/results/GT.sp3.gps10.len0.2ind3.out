-72.8445767148
